modules/core/src/norm.cpp

```diff
@@ -52,6 +52,9 @@ static const uchar popCountTable4[] =
 
 int normHamming(const uchar* a, int n, int cellSize)
 {
+    int output;
+    CALL_HAL_RET(normHamming8u, cv_hal_normHamming8u, output, a, n, cellSize);
```

Comment:
It should be just CAL_HALL, if you do not use output status.

Reply:
CALL_HAL does not work with functions returning some value, looks like CALL_HAL_RET was made specially for that case